### ISSUE TITLE
++ not recognised as mutation by const warning code

### ISSUE THREAD

**Issue by Southclaws** [Zeex/pawn#585]
This code, with `++` on `output` causes "possibly a const was intended"

```c
stock GetPlayerWoundsPerBodypart(playerid, output[7])
{
	if(!IsPlayerConnected(playerid))
		return 0;

	foreach(new i : wnd_Index[playerid])
	{
		switch(wnd_Data[playerid][i][wnd_bodypart])
		{
			case BODY_PART_TORSO:		output[0]++;
			case BODY_PART_GROIN:		output[1]++;
			case BODY_PART_LEFT_ARM:	output[2]++;
			case BODY_PART_RIGHT_ARM:	output[3]++;
			case BODY_PART_LEFT_LEG:	output[4]++;
			case BODY_PART_RIGHT_LEG:	output[5]++;
			case BODY_PART_HEAD:		output[6]++;
			default:					output[0]++;
		}
	}

	return 1;
}
```

Whereas `+= 1` does not:

```c
stock GetPlayerWoundsPerBodypart(playerid, output[7])
{
	if(!IsPlayerConnected(playerid))
		return 0;

	foreach(new i : wnd_Index[playerid])
	{
		switch(wnd_Data[playerid][i][wnd_bodypart])
		{
			case BODY_PART_TORSO:		output[0] += 1;
			case BODY_PART_GROIN:		output[1] += 1;
			case BODY_PART_LEFT_ARM:	output[2] += 1;
			case BODY_PART_RIGHT_ARM:	output[3] += 1;
			case BODY_PART_LEFT_LEG:	output[4] += 1;
			case BODY_PART_RIGHT_LEG:	output[5] += 1;
			case BODY_PART_HEAD:		output[6] += 1;
			default:					output[0] += 1;
		}
	}

	return 1;
}
```

**Comment by Daniel-Cortez** [comment]
Already fixed in #537.

**Comment by Daniel-Cortez** [comment]
Also, a duplicate of #536.
